### ISSUE TITLE
[Feature request] Keep Directory Structure Found in Watch Folder

### ISSUE THREAD

**Issue by Silun** [jlesage/docker-handbrake#289]
### Idea

Oftentimes, I am transcoding a larger number of video files I have, like archived YouTube channels, recorded Twitch streams, and the like. Those are all sorted in their respective directories. Example here:

```
Folder
---Subfolder-1
------File1
------File2
---Subfolder-2
------File3
---File4
---File5
---File6
```

Now when I FTP that over to Handbrake's watchfolder, it will find all those files and transcode them, but the directory structure in the output folder will be lost, everything is flattened directly like this:

```
output
---File1
---File2
---File3
---File4
---File5
---File6
```

Is there any way that I can make Handbrake recreate the folder structure in the output? The goal here is to be able to just drop in the new files from where I took the originals without having to manually sort them into their locations. I couldn't find anything to that effect, so if currently this cannot be done, then I am hereby requesting this feature! Loving the container overall, this would be the cherry on top for me :)

**Comment by jlesage** [comment]
Looks like you want to set `AUTOMATED_CONVERSION_OUTPUT_SUBDIR` to `SAME_AS_SRC` ?
https://github.com/jlesage/docker-handbrake#environment-variables

**Comment by Silun** [comment]
Absolutely! No idea how I missed this, I looked through the readme. Sorry and thank you :)
